activedirectory.sgnl.yaml

```diff
@@ -0,0 +1,229 @@
+      - name: pwdLastSet
+        externalId: pwdLastSet
+        type: Int64
+      - name: primaryGroupID
+        externalId: primaryGroupID
+        type: Int64
+      - name: accountExpires
+        externalId: accountExpires
+        type: Int64
+      - name: logonCount
+        externalId: logonCount
+        type: Int64
+      - name: sAMAccountName
+        externalId: sAMAccountName
+        type: String
+      - name: sAMAccountType
+        externalId: sAMAccountType
+        type: Int64
+      - name: userPrincipalName
+        externalId: userPrincipalName
+        type: String
+      - name: objectCategory
+        externalId: objectCategory
+        type: String
+      - name: dSCorePropagationData
+        externalId: dSCorePropagationData
+        type: DateTime
+  Group:
+    description: "Group Entity in Active Directory"
+    displayName: Group
+    externalId: Group
+    pageSize: 999
+    pagesOrderedById: false
+    attributes:
+      - name: dn
+        externalId: dn
+        description: distinguished Name of the user
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: cn
+        externalId: cn
+        type: String
+      - name: description
+        externalId: description
+        type: String
+      - name: distinguishedName
+        externalId: distinguishedName
+        type: String
+      - name: instanceType
+        externalId: instanceType
+        type: Int64
+      - name: whenCreated
+        externalId: whenCreated
+        type: DateTime
+      - name: whenChanged
+        externalId: whenChanged
+        type: DateTime
+      - name: uSNCreated
+        externalId: uSNCreated
+        type: Int64
+      - name: uSNChanged
+        externalId: uSNChanged
+        type: Int64
+      - name: name
+        externalId: name
+        type: String
+      - name: sAMAccountName
+        externalId: sAMAccountName
+        type: String
+      - name: sAMAccountType
+        externalId: sAMAccountType
+        type: Int64
+      - name: groupType
+        externalId: groupType
+        type: Int64
+      - name: objectCategory
+        externalId: objectCategory
+        type: String
+      - name: dSCorePropagationData
+        externalId: dSCorePropagationData
+        type: DateTime
+  GroupMember:
+    description: "Group Member Entity in Active Directory"
+    displayName: GroupMember
+    externalId: GroupMember
+    pageSize: 999
+    pagesOrderedById: false
+    attributes:
+      - name: dn
+        externalId: dn
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: groupDistinguishedName
+        externalId: groupDistinguishedName
+        type: String
+        indexed: true
+      - name: memberDistinguishedName
+        externalId: memberDistinguishedName
+        type: String
+        indexed: true
+
+relationships:
+  UserMember:
+    name: Member
+    displayName: User Member
+    fromAttribute: GroupMember.memberDistinguishedName
```

Comment:
nit: You can use `attributeAlias` to have more meaningful names if attributeExternalId is unclear